README.md

```diff
@@ -11,7 +11,7 @@ Summary: Sphinx is a tool that makes it easy to create intelligent and beautiful
 
 Sphinx is a tool that makes it easy to create intelligent and beautiful documentation,
 written by Georg Brandl and licensed under the BSD license.
-It was originally created for the new Python documentation, and it has excellent
+It was originally created for the new Python documentation and it has excellent
```

Comment:
I guess this is coming from commit ( https://github.com/conda-forge/sphinx-feedstock/commit/b146814829b6c9ec749b732b465ec71e0372b080 ), which is fine. We can revert the recipe change and re-render if we need to go back.